env_support/cmake/custom.cmake

```
@@ -46,37 +46,54 @@ if(LV_CONF_SKIP)
   target_compile_definitions(lvgl PUBLIC LV_CONF_SKIP=1)
 endif()
 
+
+
+
 # Include root and optional parent path of LV_CONF_PATH
 target_include_directories(lvgl SYSTEM PUBLIC ${LVGL_ROOT_DIR} ${LV_CONF_DIR} ${CMAKE_CURRENT_BINARY_DIR})
 
+if(LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
+  message(FATAL_ERROR "LV_CONF_BUILD_DISABLE_THORVG_INTERNAL is deprecated, use CONFIG_LV_USE_THORVG_INTERNAL instead")
+endif()
 
-if(NOT LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
+if(LV_CONF_BUILD_THORVG_INTERNAL)
     add_library(lvgl_thorvg ${THORVG_SOURCES})
     add_library(lvgl::thorvg ALIAS lvgl_thorvg)
     target_include_directories(lvgl_thorvg SYSTEM PUBLIC ${LVGL_ROOT_DIR}/src/libs/thorvg)
     target_link_libraries(lvgl_thorvg PUBLIC lvgl)
+    target_compile_definitions(lvgl_thorvg PUBLIC LV_USE_THORVG_INTERNAL=1 LV_USE_THORVG_EXTERNAL=0)
```

```suggestion
    target_compile_definitions(lvgl_thorvg PUBLIC LV_USE_VECTOR_GRAPHIC=1 LV_USE_THORVG_INTERNAL=1 LV_USE_THORVG_EXTERNAL=0)
```

It should make the CI pass.